### DISCUSSION TITLE
Wolfi Base Package

### DISCUSSION THREAD

**Discussion by shresthaujjwal** [wolfi-dev/os#10160] [Q&A] [answered]
Where can I find more information on wolfi-base package. I found the source code for wolfi-base layout and key but couldn't find any details on wolfi-base. Can you please help me find this information. Thanks in advance

**Comment by shresthaujjwal** [comment]
any update on this ?

**Comment by kranurag7** [comment, marked as answer]
This is the base-layout of the wolfi image. 
https://github.com/wolfi-dev/os/blob/main/wolfi-baselayout.yaml 

- this is what is included as part of wolfi-base package. 
https://github.com/wolfi-dev/os/blob/main/wolfi-base.
- Read more about it here. 
https://edu.chainguard.dev/chainguard/chainguard-images/reference/wolfi-base/

**Comment by amouat** [comment]
Thanks @kranurag7
